transform/startf/testdata/load_ds.star

```diff
@@ -1,6 +1,6 @@
 load('assert.star', 'assert')
 
-movies = load_dataset("peer/movies")
+movies = load_version("peer/movies")
```

Comment:
Is this function staying as `load_version`. If so, it reads weird to pass it the ref to a dataset, without any version information. Would rather keep it as `load_dataset`.

Reply:
ooops, should totally be `load_dataset`, will change